### DISCUSSION TITLE
这么好的miniFOC为什么没多少人讨论?

### DISCUSSION THREAD

**Discussion by wocut** [ZhuYanzhen1/miniFOC#9] [General]
我在跟着制造这个板是单个遇到很多问题啊！我只会用CubeMX 和keil开发stm32,能不能用同样的工具开发这个miniFOC?但这个GD32F130G6U6TR的芯片对应stm32的什么型号芯片？

**Comment by ZhuYanzhen1** [comment]
感谢支持，此项目可以用keil开发，但是需要做一些移植。具体在keil上开发的方法直接参考GD32官网给出的keil例程即可。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年2月10日(周四) 晚上8:48
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [ZhuYanzhen1/miniFOC] 这么好的miniFOC为什么没多少人讨论? (Discussion #9)




 
我在跟着制造这个板是单个遇到很多问题啊！我只会用CubeMX 和keil开发stm32,能不能用同样的工具开发这个miniFOC?但这个GD32F130G6U6TR的芯片对应stm32的什么型号芯片？
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
